downstream/modules/platform/con-aap-upgrade-prereq.adoc

```diff
@@ -0,0 +1,147 @@
+. Locate the public server pool section and modify it to include the appropriate NTP server addresses. Only one server is required, but three is recommended. The `iburst` option is added to speed up the time that it takes to properly sync with the servers:
++
+----
+# Use public servers from the pool.ntp.org project.
+# Please consider joining the pool (http://www.pool.ntp.org/join.html).
+server <ntp-server-address> iburst
+----
+. Save changes within the `/etc/chrony.conf` file.
+. Start the host and enable the `chronyd` daemon:
++
+----
+# systemctl --now enable chronyd.service
+----
+. Verify the `chronyd` daemon status:
++
+----
+# systemctl status chronyd.service
+----
+
+.Attaching Red Hat subscription on all nodes
+{PlatformName} requires that you have valid subscriptions attached to all nodes. You can verify if your current node has a Red Hat subscription by running the following command:
+----
+# subscription-manager list --consumed
+----
+If there is not a Red Hat subscription attached to the node, see the https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_installation_guide/planning-installation#proc-attaching-subscriptions_planning[attaching your {PlatformNameShort} subscription] guide on how to do so.
+
+.Creating non-root user with sudo privileges
+Prior to your upgrade of {PlatformNameShort}, it is recommended to create a non-root user with sudo privileges for the deployment process. This user is used for:
+
+* SSH connectivity.
+* Passwordless authentication during installation.
+* Privilege escalation (sudo) permissions.
+
+The following examples uses `ansible` to name this user. On all nodes used in the upgraded {PlatformNameShort} cluster, create a non-root user named `ansible` and generate an ssh key:
+
+. Create a non-root user:
++
+----
+# useradd ansible
+----
+. Set a password for your user:
++
+----
+# passwd ansible <1>
+Changing password for ansible.
+Old Password:
+New Password:
+Retype New Password:
+----
+<1> Replace `ansible` with the non-root user from step 1, if using a different name
+. Generate an `ssh` key as the user:
++
+----
+$ ssh-keygen -t rsa
+----
+. Disable password requirements when using `sudo`:
++
+----
+# echo "ansible ALL=(ALL) NOPASSWD:ALL" | sudo tee -a /etc/sudoers.d/ansible
+----
+
+.Copying SSH keys to all nodes
+WIth the `ansible` user created, copy the `ssh` key to all the nodes used in the upgraded {PlatformNameShort} cluster. This ensures that when the {PlatformNameShort} installation runs, it can `ssh` to all the nodes without a password:
+----
+$ ssh-copy-id ansible@node-1.example.com
+----
+NOTE: If running within a cloud provider, you may need to instead create an `~/.ssh/authorized_keys` file containing the public key for the `ansible` user on all your nodes and set the permissions to the `authorized_keys` file to only the owner (`ansible`) having read and write access (permissions 600).
```

Comment:
IBM Style says to use "may" to indicate permission and "might" to indicate possibility. Therefore, I suggest changing "you may need..." to "you might need..." here.